extension-config.json

```diff
@@ -209,6 +209,33 @@
         "path": "/v1/shop/%(shopId)s/checkout_web_checkout_shopify?parsed=true",
         "key": "value.frontend.alias"
       }
+    },
+    "redisClientShopNumber": {
```

Comment:
might not need this configuration. I think this is available from context.meta.appId

Reply:
Fixed